controller/controller.go

```diff
@@ -29,6 +30,8 @@ func NewController(uc usecase.Usecaseinterface) *PokemonController {
 
 }
 
+/// in this method recives the parameters from the request , get the vaiable name from the  request , verify is an string
+/// then  search for given pokemon name and send the response back in case the pokemon exist
```

Comment:
Be consistent; only two `/` per comment.

Remember, the comment should start with the name of the element you are documenting.

Fix this here and all other instances.
```suggestion
// GetPokemonByName receives the parameters from the request, get the variable name from it, verify if it's an string
// then search for given pokemon name and send the response back in case the pokemon exists.
```